### DISCUSSION TITLE
Welcome to tuya-local Discussions!

### DISCUSSION THREAD

**Discussion by make-all** [make-all/tuya-local#466] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jpeters001** [comment]
Thank You, I installed this and was able to get all my Tuya devices running on it including a water timer that previously i could only run on the app and not through homeassistant at all! Whats more I was able to configure all the devices automatically using the prefilled information for each device.  Now I would like to know what is the next step, I have Tuya and Tuya Local both running and so all my lights are showing up twice in homeassistant. Do I disable the Tuya integration and only keep Tuya Local? Also, is it smart to block internet access now that they are working locally, and if so does anyone know of any good info for the best practices to do this for a noob like me? once again, a giant Thank You to all those that have worked to make this integration.
